INSTALL.md

```diff
@@ -84,13 +84,20 @@ If you are interested in packaging, documenting, or contributing otherwise, plea
 
 ## Docker images <a name="docker"></a>
 You can run Shogun in [our own cloud](cloud.shogun.ml) or set up your own using our
-[Docker images](https://hub.docker.com/r/shogun/shogun-dev/) as:
+[Docker images](https://hub.docker.com/r/shogun/shogun/) as:
 
-    sudo docker pull shogun/shogun
-    sudo docker run -it shogun/shogun bash
+    sudo docker pull shogun/shogun:master
+    sudo docker run -it shogun/shogun:master bash
 
 We offer images for both the latest release and nightly development builds.
 
+for developer version [Docker images](https://hub.docker.com/r/shogun/shogun-dev/)
```

Comment:
Make it: "For the [developer version](https://hub.docker.com/r/shogun/shogun-dev/) version, replace `shogun/shogun:master` with `shogun/shogun-dev`.

(No need to show a full line command again)